pytest/lib/cluster.py

```diff
@@ -475,6 +475,9 @@ def get_changes_in_block(self, changes_in_block_request):
                              changes_in_block_request)
 
     def get_changes(self, changes_request):
+        return self.json_rpc('changes', changes_request)
+
+    def get_experimental_changes(self, changes_request):
```

Comment:
Can you also mark this one as deprecated? comment + some python-y way if it exists would be nice